backend/src/main/java/zipgo/petfood/domain/Reviews.java

```diff
@@ -20,27 +28,100 @@
 @AllArgsConstructor
 public class Reviews {
 
+    private static final int PERCENTAGE = 100;
+    private static final double DEFAULT_PERCENTAGE = 0.0;
+
     @Default
     @OneToMany(mappedBy = "petFood")
-    private List<zipgo.review.domain.Review> reviews = new ArrayList<>();
+    private List<Review> reviews = new ArrayList<>();
 
     public double calculateRatingAverage() {
         double average = reviews.stream()
                 .mapToInt(review -> review.getRating())
                 .average()
-                .orElse(0);
+                .orElse(DEFAULT_PERCENTAGE);
         return roundToOneDecimal(average);
     }
 
     private double roundToOneDecimal(double average) {
         return Math.round(average * 10) / 10.0;
     }
 
+    public int getRatingPercentage(int rating) {
+        Map<Integer, Long> ratingAndCount = getRatingAndCount();
+        long count = ratingAndCount.getOrDefault(rating, 0L);
+        return calculateRatingPercentage(reviews.size(), count);
+    }
+
+    private Map<Integer, Long> getRatingAndCount() {
+        return reviews.stream()
+                .collect(Collectors.groupingBy(
+                        Review::getRating,
+                        Collectors.counting()
+                ));
+    }
+
+    private int calculateRatingPercentage(int totalReviews, long count) {
+        if (totalReviews == 0) {
+            return 0;
+        }
+        return (int) ((count * PERCENTAGE) / totalReviews);
+    }
+
+    public int getTastesPercentage(String tastePreference) {
+        Map<String, Long> tastesAndCount = getTastesAndCount();
+        long count = tastesAndCount.getOrDefault(tastePreference, 0L);
+        return TastePreference.getDistributionPercentage(reviews.size(), count);
+    }
+
+    private Map<String, Long> getTastesAndCount() {
+        return reviews.stream()
+                .collect(groupingBy(
+                        review -> review.getTastePreference().name(),
+                        counting()
+                ));
+    }
+
+    public int getStoolsConditionPercentage(String stoolCondition) {
```

Comment:
```suggestion
public int getStoolsConditionPercentage(StoolCondition stoolCondition) {
```
질문) 메서드 사용자를 고려한다면 이렇게 매개변수를 제한해주는 게 좋다고 생각하는데, 베베가 이렇게 구현한 이유가 궁금합니다!

Reply:
오.. 사실 enum으로 받는 생각을 못하고 있었는데.. 어쩐지 계속 .name()함수를 사용하게 되더라구요.. 반영하겠습니다!